lib/login/login_page.dart

```diff
@@ -8,10 +9,103 @@ class LoginPage extends StatelessWidget {
   Widget build(BuildContext context) {
     return Scaffold(
       body: Center(
-        child: LoginButton(
-          onLoginClicked: () {},
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          // ignore: prefer_const_literals_to_create_immutables
+          children: [
+            Padding(
+              padding: const EdgeInsets.fromLTRB(40, 0, 40, 0),
+              child: TextField(
+                obscureText: true,
+                decoration: InputDecoration(
+                  icon: const Icon(
+                    Icons.person,
+                    color: Color(0xffc6c6c6),
+                  ),
+                  hintText: 'Username',
+                  hintStyle: const TextStyle(
+                    color: Color(0xffc6c6c6),
+                  ),
+                  enabledBorder: myUnderlineBorder(),
+                ),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.fromLTRB(40, 0, 40, 40),
+              child: TextField(
+                obscureText: true,
+                decoration: InputDecoration(
+                  icon: const Icon(
+                    Icons.lock,
+                    color: Color(0xffc6c6c6),
+                  ),
+                  hintText: 'Password',
+                  hintStyle: const TextStyle(
+                    color: Color(0xffc6c6c6),
+                  ),
+                  enabledBorder: myUnderlineBorder(),
+                ),
+              ),
+            ),
+            LoginButton(
+              onLoginClicked: () {},
+            ),
+            Padding(
+              padding: const EdgeInsets.fromLTRB(
+                40,
+                40,
+                20,
+                0,
+              ),
```

Comment:
```suggestion
              padding: const EdgeInsets.only(
                right: 40,
                left: 40,
                top: 20,
              ),
```

Reply:
You should consider using a horizontal padding of 40 around the whole view. What do you think?

Reply:
Yes, you're right. That would make more sense.